README.md

```diff
@@ -20,12 +20,36 @@ Any exceptions to this are noted in the individual `README.md` files.
 
 ## Installation
 
-TBD
+The repository layout structure is as defined by [the specification](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-layout.md). The `api`
```

Comment:
I think we need to decide on a few practical things. When you create a package, what is the actual installation / deployment layout:

- /usr/local/lib
- /usr/local/include/opentelemetry - for API?
- /usr/lcoal/include/opentelemetry/sdk - fo SDK developers?

# Why this needs to be described

Many packaging tools, e.g. `vcpkg` or `deb` would prefer you to have a clear-cut definition of what are the ship'able artifacts, even if include headers. Usually the layout for C++ projects requires a single collated directory. In our case it is two directories: one for customers `api` and another for devs, `sdk`... Are we shipping two separate packages, such as `opentelemetry` and `libopentelemetry-dev` - for developers?

Reply:
I am now confused here. As per my understanding, and also explicitly specified in this PR, we are only going to have source-only releases in same layout as the repo ( same as provided by github release ui by default). Which would mean, it would be responsibility of developers to decide how they would like to package this and distribute further - which should ideally be the structure as you mentioned above. Do you mean, even though we provide source-only releases, there should be recommendation for developer how to package them ?

Reply:
In order to consume the package with tooling, like `vcpkg` (which builds the release for you from source and installs it for you), we need to identify the layout of how what we expect to get as a result of the build. In other words, if you build from source and invoke `make install` - what is the expected outcome?

Reply:
Ok, thanks for clarification. I will add expected deployment layout structure from packaging tools propspective in the document. 
As of now, we are installing api headers at `/usr/local/include/opentelemetry` for cmake builds. We can extended that as suggested above:

```
/usr/local/lib  -> For libraries
/usr/local/include/opentelemetry -> for api headers
/usr/local/include/opentelemetry/sdk -> for sdk headers
```

--
Are we shipping two separate packages, such as opentelemetry and libopentelemetry-dev - for developers?
 --
We are not shipping anything :) Although same recommendation on the final library/header deployment structure would apply for source only package like `libopentelemetry-dev` once it is `installed` and build from `/usr/local/src`.